docs/gallery/External_Resources.py

```diff
@@ -63,14 +96,12 @@
 ###############################################################################
 # Using the add_ref method with a field
 # ------------------------------------------------------
-# In the above example, field was empty because the data of the Data object was
-# being associated with a resource. The Data object may also have attributes
-# that you would like to associate with resources.
-# Similarly, a Container object may have several datasets and attributes that
-# you would like to associate with resources.
-# To disambiguate between these different fields, you can set the 'field' keyword
-# argument corresponding to the name of the dataset or attribute containing the
-# values being associated with a resource. For example:
+# In the above example, ``field`` was empty because the data of the ``container``
+# data object itself was being associated with a resource. However, rather than
+# the ``container`` data object itself, you may instead want to associated external
+# resources with datasets or attributes contained in the data object.
+# that you would like to associate with resources. To disambiguate between
+# these different fields, you can set the 'field' keyword.
```

Comment:
```suggestion
# In the above example, the ``field`` keyword argument was empty because the data
# of the :py:class:`~hdmf.container.Data` object passed in for the ``container``
# argument was being associated with a resource. However, you may want to associate
# an attribute of a :py:class:`~hdmf.container.Data` object with a resource or
# a dataset or attribute of a :py:class:`~hdmf.container.Container` object with
# a resource. To disambiguate between these different fields, you can set the
# 'field' keyword.
```